inlet/bioris/config.go

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+// Package bioris provides an interface to hydrate flow from biorouting ris
+package bioris
+
+import (
+	"reflect"
+
+	"akvorado/common/helpers"
+
+	"github.com/mitchellh/mapstructure"
+)
+
+// Configuration describes the configuration for the BioRIS component.
+type Configuration struct {
+	// RISInstances holds the different ris connections
+	RISInstances []RISInstance
+}
+
+// RISInstance stores the connection details of a single RIS connection
+type RISInstance struct {
+	GRPCurl    string
+	GRPCSecure bool
+	VRFId      uint64
+	VRF        string
+}
+
+// DefaultConfiguration represents the default configuration for the
+// RISInstance component. Without connection, the component won't report
+// anything.
+func DefaultConfiguration() Configuration {
+	return Configuration{
+		RISInstances: []RISInstance{},
+	}
+}
+
+// ConfigurationUnmarshallerHook normalize BioRIS configuration
+func ConfigurationUnmarshallerHook() mapstructure.DecodeHookFunc {
+	return func(from, to reflect.Value) (interface{}, error) {
+		if from.Kind() != reflect.Map || from.IsNil() || to.Type() != reflect.TypeOf(Configuration{}) {
+			return from.Interface(), nil
+		}
+
+		return from.Interface(), nil
+	}
+}
```

Comment:
This can be removed.

Reply:
I've cleaned this up and added a rudimentary test based on the other inlet config tests.